src/components/Card.js

```diff
@@ -15,22 +25,32 @@ export default class Card {
 
   //event listeners for cards
   _setEventListeners() {
-    this._likeButton.addEventListener("click", this._likeCard);
-    this._deleteButton.addEventListener("click", this._removeCard);
+    this._likeButton.addEventListener("click", () => {
+      this.handleLikeClick(this);
+      this._checkedIfLiked();
+    });
+    this._deleteButton.addEventListener("click", () => {
+      this.handleTrashClick(this);
+    });
     this._cardImage.addEventListener("click", this.handleImageClick);
   }
 
-  _likeCard(e) {
-    e.target.classList.toggle("card__like-icon_clicked");
-  }
+  _checkedIfLiked = () => {
```

Comment:
Needs correction: please rename this method to something like `renderLikes` as it doesn't only check the likes state it renders their view